netcon/inetfile/Chttpget.cpp

```diff
@@ -211,7 +211,12 @@ void ChttpGet::PrepSocket(char *URL) {
     return;
   }
 
-  make_nonblocking(m_DataSock);
```

Comment:
make_nonblocking fails here? I thought we fixed it

Reply:
`make_nonblocking()` pulls into DLL unnecessary dependencies - networking, ddio and friend of these.

Reply:
got it, maybe make_nonblocking should go somewhere more accessible